avocado/core/replay.py

```diff
@@ -41,6 +42,9 @@ def record(args, logdir, mux, urls=None):
     if urls:
         with open(path_urls, 'w') as f:
             f.write('%s' % urls)
+        if url_archive:
+            path_archive = os.path.join(replay_dir, 'urls_archive.tar.gz')
+            archive.create(path_archive, urls, absolute_path = True)
```

Comment:
Well, the replay feature was born to re-run with the same options a test that can be (or not) changed so we can run always the latest version of the test with the same avocado configurations.
Anyway, I see a value in 'freeze' the test to replay it as it was in the original job run. Not sure if we should take care of it, since git is out there and it is a great versioning system. You can always have versions of your test in git and replay different versions of it with the same avocado options, provided by the replay feature.
Anyway, this seems incomplete, since I see you recording the test but not retrieving it.